### ISSUE TITLE
Chapitre 2: hands on ML

### ISSUE THREAD

**Issue by dfshr8** [ageron/handson-ml#230]
Bonjour Mr Géron et merci infiniment pour votre merveilleux livre !
Lors de la création du train/test set vous proposez de partitionner le label (valeur continue du prix médian des maisons) afin de réaliser un partitionnement stratifié: cette idée est géniale et je ne comprends pas pourquoi elle n'est pas réutilisée lorsque l'on évalue le modèle par validation croisée (lorsque le label est une catégorie sklearn le fait automatiquement mais pas lorsque le label est continue)?

**Comment by ageron** [comment]
Bonjour @dfshr8 ,
Désolé, j'ai manqué votre message. En effet, si l'on fait un partitionnement stratifié, il vaut mieux le faire pour l'ensemble des jeux de données, et pour la validation croisée, vous avez raison. Je ne l'ai pas fait par souci de simplicité (une grande amie de la paresse), mais aussi car l'exemple du partitionnement stratifié était avant tout une petite illustration du principe des "splitters".
J'espère que vous avez apprécié le reste du livre!
Aurélien
